.github/workflows/check-3rd-party.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+FILE=$1
+status=0
+
+for line in `sed -ne 's/[[:space:]-]*uses:[[:space:]]*//p' $1 | sed -e 's/\s*#.*$//'`; do
+  author=`echo $line | awk -F/ '{print $1}'`
+  if [ $author == "actions" ]; then continue; fi
```

Comment:
since this is bash, might as well use [[ instead of [, which has fewer footguns :)